app/src/main/java/com/gmail/mxwild/newhabit/di/RetrofitModule.kt

```diff
@@ -0,0 +1,54 @@
+package com.gmail.mxwild.newhabit.di
+
+import com.gmail.mxwild.newhabit.BuildConfig
+import com.gmail.mxwild.newhabit.api.TheMovieDbApi
+import com.gmail.mxwild.newhabit.services.ApiKeyInterceptor
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class RetrofitModule {
```

Comment:
Do you consume anything except API out of the module? Why provide the rest if it is not used?